test/.gitignore

```diff
@@ -0,0 +1 @@
+newman
```

Comment:
I suggest to organize the e2e test into "e2e" folder, so that `acceptance` / `unit` / `e2e` test can be well managed under "test", also in future we may have multiple e2e test process. Place all of them under "test/" may get mixed-up.

Reply:
It is under test folder of top directory, so it is not mixed with the unit test of each app.

Reply:
ok